src/main/java/com/squareup/javapoet/Util.java

```diff
@@ -32,6 +38,72 @@
   private Util() {
   }
 
+  private static final Modifier DEFAULT;
+  /**
+   * Used to detect conflicting modifiers, maps from a modifier to the
+   * conflicting modifier list.
+   */
+  private static final Map<Modifier, Set<Modifier>> CONFLICTING_MODIFIER_MAP;
+  static {
+    Modifier def = null;
+    try {
+      def = Modifier.valueOf("DEFAULT");
+    } catch (IllegalArgumentException expected) {
```

Comment:
nit: expected –> ignored 

(expected: this should happen 100% of the time. ignored: I don't care if this happens)